include/openthread/border_agent.h

```diff
@@ -256,8 +256,11 @@ otError otBorderAgentSetEphemeralKey(otInstance *aInstance,
  *
  * @param[in] aInstance    The OpenThread instance.
  *
+ * @retval OT_ERROR_NONE           Successfully cleared the ephemeral key.
+ * @retval OT_ERROR_INVALID_STATE  Border Agent is connected to an external commissioner.
```

Comment:
If we add this error, we might also want to update the description above to reflect it.

Additionally, it raises the question of whether we should return an error if this function is called when no ephemeral key is in use (either no key is set, or a previously set key has timed out and expired).

Reply:
updated the description for the returned errors.

I think the desired end state of this method is that there is no active ephemeral key, so `OT_ERROR_NONE ` seems appropriate for the case that there is no ephemeral key to clear.  Thoughts?